### ISSUE TITLE
on_payment_verified

### ISSUE THREAD

**Issue by dera1992** [gbozee/pypaystack#14]
Please, i cant get "on_payment_verified" function to work after payment

**Comment by Fawaz441** [comment]
Yes, the verify_payment function where the signal is being sent requires 'request=request'

**Comment by dera1992** [comment]
ok, thanks for the reply. How can i add 'request=request'. Thanks.

**Comment by austinogiza** [comment]
I also have the same issue, how can I add request=request for the user @Fawaz441

**Comment by ajadi473** [comment]
Thanks for the tutorial, it was much helpful. 

However, can i have a clue as to how to make the details gets stored in the transaction history table or where can i write the save function?

**Comment by austinogiza** [comment]
hello @Fawaz441 , can you help out

**Comment by Fawaz441** [comment]
![pay](https://user-images.githubusercontent.com/48877023/82897802-1bc02a00-9f50-11ea-906c-00127d33fd29.JPG)
in paystack/views.py

**Comment by ajadi473** [comment]
> ![pay](https://user-images.githubusercontent.com/48877023/82897802-1bc02a00-9f50-11ea-906c-00127d33fd29.JPG)
> in paystack/views.py

Thanks for this, 

I got it working to this point unfortunately after a successful payment, i couldn't output the any json data  just as it is in this picture attached.

What should I do  or is it a webhook thing?
![paymentsuccess](https://user-images.githubusercontent.com/6432574/82898288-e6680c00-9f50-11ea-8022-7eada2624c48.png)

**Comment by Fawaz441** [comment]
You have json installed?

**Comment by ajadi473** [comment]
yes i do

**Comment by ajadi473** [comment]
could it be because i am using the test api secret and public keys from paystack and not the live api  public keys

**Comment by austinogiza** [comment]
@Fawaz441 is it possible if I can contact you bro

**Comment by Fawaz441** [comment]
Sure you can get me on WhatsApp 09082150457

**Comment by bgreatfit** [comment]
> @Fawaz441 is it possible if I can contact you bro, [austinogiza@gmail.com](mailto:austinogiza@gmail.com)..

Hi, I am having a namespace issue
